tools/rosbag/src/rosbag/rosbag_main.py

```diff
@@ -269,6 +271,12 @@ def play_cmd(argv):
     if options.topics or options.pause_topics:
         cmd.extend(['--bags'])
 
+    if options.rate_control_topic:
+        cmd.extend(['--rate-control-topic', str(options.rate_control_topic)])
+
+    if options.rate_control_max_delay:
+        cmd.extend(['--rate-control-max-delay', str(options.rate_control_max_delay)])
+
```

Comment:
@jihoonl Inserting this between `--bags` and `args` broke the feature to pass topics or pausing topics. See #1156 for the necessary fix.